firmware/boron-ins-fsm/src/im21door.cpp

```diff
@@ -164,15 +164,17 @@ void threadBLEScanner(void *param) {
   while(true){
```

Comment:
Can you please add a line in the CHANGELOG about this addition?

Reply:
I added a line in the changelog for these changes in my latest commit